From: hyperv-linux-gpup maintainers
Subject: [PATCH 5/5] dxgkrnl: use public task comm helper

--- a/drivers/hv/dxgkrnl/dxgvmbus.c
+++ b/drivers/hv/dxgkrnl/dxgvmbus.c
@@ -689,7 +689,7 @@ int dxgvmb_send_create_process(struct dxgprocess *process)
 	command->process_id = process->pid;
 	command->linux_process = 1;
 	s[0] = 0;
-	__get_task_comm(s, WIN_MAX_PATH, current);
+	get_task_comm(s, current);
 	for (i = 0; i < WIN_MAX_PATH; i++) {
 		command->process_name[i] = s[i];
 		if (s[i] == 0)
